### ISSUE TITLE
Members Page Group Tab Copy Updates

### ISSUE THREAD

**Issue by zakhap** [hicommonwealth/commonwealth#5611]
## Description + Acceptance Criteria

 **Members page, Groups Tab (empty state)**
- [x] "[community] does not have any groups." ⇒ Capitalize [community] name. Currently, not capitalized.
- [x] "Create a group to gate discussion topics" should become ⇒ “Admins can create groups to gate discussion topics”
<img width="627" alt="Screen Shot 2023-11-07 at 2 05 59 PM" src="https://github.com/hicommonwealth/commonwealth/assets/16248081/4cfcb439-bcb6-4769-89e5-1e4d76a85252">

**Members page, Groups Tab (non-empty state)**
- [x] While viewing the Groups + their requirements, for EVM + ERC20 Requirements, we should convert the “amount” integer to decimal (by 18 is standard for EVM + ERC20).
    - For example, “80000” ⇒   80000*10^-18 ⇒ .000000000000008.
<img width="542" alt="Screen Shot 2023-11-07 at 2 10 29 PM" src="https://github.com/hicommonwealth/commonwealth/assets/16248081/c498a48e-fc1a-4bb0-b767-982f7590603f">

## Project Owner

@zakhap

**Comment by markiscommon** [comment]
fire

**Comment by mzparacha** [comment]
> While viewing the Groups + their requirements, for EVM + ERC20 Requirements, we should convert the “amount” integer to decimal (by 18 is standard for EVM + ERC20).
For example, “80000” ⇒ 80000*10^-18 ⇒ .000000000000008.

Should we also do something in case of erc721/cosmos-token? @zakhap

**Comment by zakhap** [comment]
Nothing for ERC721. 

Cosmos is possibly more convoluted so we don't need to worry about it at this time. We will do a fast follow research + solutioning to make sure our display and input makes sense in the Cosmos context. 

tldr: nothing needs to change for the display for ERC721 or Cosmos-tokens at this time.
